From 8b04780f85fb75246e09ecf004b116f446794251 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Sun, 29 Sep 2019 08:55:20 +0200
Subject: [PATCH 1/8] arm: dts: sunxi: Orange Pi Zero - simplification

Do not acitvate ohci0 - USB OTG do not need this.
Remove already disabled resources.
Normalize names of LED nodes.
Remove (disabled) flash configuration.

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
index f19ed981da9d..8d43826c1c27 100644
--- a/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
@@ -68,13 +68,13 @@ chosen {
 	leds {
 		compatible = "gpio-leds";
 
-		pwr_led {
+		pwr {
 			label = "orangepi:green:pwr";
 			gpios = <&r_pio 0 10 GPIO_ACTIVE_HIGH>;
 			default-state = "on";
 		};
 
-		status_led {
+		status {
 			label = "orangepi:red:status";
 			gpios = <&pio 0 17 GPIO_ACTIVE_HIGH>;
 		};
@@ -154,45 +154,16 @@ xr819: sdio_wifi@1 {
 	};
 };
 
-&ohci0 {
-	status = "okay";
-};
-
 &ohci1 {
 	status = "okay";
 };
 
-&spi0 {
-	/* Disable SPI NOR by default: it optional on Orange Pi Zero boards */
-	status = "disabled";
-
-	flash@0 {
-		#address-cells = <1>;
-		#size-cells = <1>;
-		compatible = "mxicy,mx25l1606e", "winbond,w25q128";
-		reg = <0>;
-		spi-max-frequency = <40000000>;
-	};
-};
-
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pa_pins>;
 	status = "okay";
 };
 
-&uart1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&uart1_pins>;
-	status = "disabled";
-};
-
-&uart2 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&uart2_pins>;
-	status = "disabled";
-};
-
 &usb_otg {
 	dr_mode = "peripheral";
 	status = "okay";
